pandas/tests/io/test_gcs.py

```diff
@@ -73,6 +73,7 @@ def test_to_read_gcs(gcs_buffer, format):
         df1.to_csv(path, index=True)
         df2 = read_csv(path, parse_dates=["dt"], index_col=0)
     elif format == "excel":
+        pytest.importorskip("tinycss2")
```

Comment:
This test seems to be exporting a `DataFrame` to Excel, not a `Styler`. Do the changes implemented here require a user to have `tinycss2` installed even if they are just exporting a `DataFrame` to Excel without even bothering with adding styles?

Reply:
The way the excel writer is implemented there really is no such thing as a truly unstyled output. The regular to_excel function adds a bunch of borders and bold headers. Since there are multiple excel output libraries supported, these borders and other things are added as CSS in the more abstract layers of code. That CSS then gets merged with the user styles when using a Styler. But there is always some CSS being processed when writing to excel.

Reply:
I think the question was meant more like: would users need to install a new dependency to use to_excel even if they don’t set user configured styles?

Reply:
Sorry my answer wasn't clear, that was the question I was trying to answer. 

The normal to_excel function uses CSS under the hood for styling. So yes all excel users would need this additional dependency.